### ISSUE TITLE
Enabling secure forward prevents sending logs to builtin elasticsearch

### ISSUE THREAD

**Issue by rmacian** [openshift/origin-aggregated-logging#1037]
I have followed the http://v1.uncontained.io/playbooks/operationalizing/secure-forward-splunk.html guide to send logs to Splunk on two different environments.  I can ingest logs to splunk but then can't be seen in Elasticsearch.

I do not understand because the file is included under the @type copy 

I want to mantain the logs on ES while I send some of them to Splunk

**Comment by mshahat** [comment]
@rmacian i am working on a similar approach, could we get in touch to possibly leverage some of the work you're doing?  Cheers,

**Comment by jlsalmon** [comment]
I am also facing this issue. Both the docs and the fluent config suggest that the logs should go to both the secure forward endpoint and to elasticsearch.

**Comment by jlsalmon** [comment]
I seem to have fixed this by editing the fluentd configmap, wrapping the secure_forward plugin config with `<store>` tags:

```
kind: ConfigMap
apiVersion: v1
data:
  secure-forward.conf: |
    <store>
        @type secure_forward
        ...
    </store>
  ...
```

**Comment by jlsalmon** [comment]
Seems to be already fixed in f3d999444947fe34545f374a49a0db30262c833e
